src/wp-includes/admin-bar.php

```diff
@@ -198,6 +201,18 @@ function wp_admin_bar_wp_menu( $wp_admin_bar ) {
 			'href'   => __( 'https://wordpress.org/support/forum/requests-and-feedback' ),
 		)
 	);
+
+	if ( $contribute_url ) {
+		// Add contribute link.
+		$wp_admin_bar->add_node(
+			array(
+				'parent' => 'wp-logo-external',
```

Comment:
What do you think about moving this to the top section? The rest of the links in the current group are external to WordPress.org.

<img width="208" alt="Screenshot 2023-07-11 at 10 32 52 AM" src="https://github.com/WordPress/wordpress-develop/assets/541093/057e5716-5c6b-4223-acb1-f8ef5ad43de7">

```suggestion
				'parent' => 'wp-logo',
```